db/migrate/20221213171103_doc_indexes.rb

```diff
@@ -4,9 +4,8 @@ def change
     remove_index :doc_statuses, [:doc_profile_id, :doc_facility_id]
     
     add_index :doc_sentences, [:doc_profile_id, :sentence_id] ,unique:true
-    add_index :doc_statuses, [:doc_profile_id, :doc_facility_id], unique:true
```

Comment:
this extra index was left on here so the migrations couldn't run
(since the combination of :doc_profile_id and :doc_facility_id aren't unique in the production database)